assets/js/util/apolloClient.js

```diff
@@ -24,7 +26,25 @@ const authLink = setContext((_, { headers }) => {
   }
 })
 
-const authHttpLink = authLink.concat(httpLink)
+const authErrorLink = onError(({ networkError, operation: { operationName }}) => {
+  if (networkError.statusCode == 404) {
```

Comment:
not sure if listening for 404s is correct here, or if I should make it so that the device/channel/gateway show query should return no errors but return a successful response with a message that says none was found. Let me know what you think